homeassistant/components/lyric/sensor.py

```diff
@@ -90,6 +90,22 @@ def get_setpoint_status(status: str, time: str) -> str | None:
                         device,
                     )
                 )
+            if device.indoorHumidity:
+                entities.append(
+                    LyricSensor(
+                        coordinator,
+                        LyricSensorEntityDescription(
+                            key=f"{device.macID}_indoor_humidity",
```

Comment:
Sifd note for the future: Please refactor and don't include the device mac ID in the entity description key. Just use the suffix. The entity unique_id property can be changed to compose the unique_id with the device mac ID and the entity description key.

We want entity descriptions to be declared as constants at the module level so they are easier to read.